### PR TITLE
Initial CircleCI config

### PR DESCRIPTION
**Pull request by lucafavatella**
Cherry-picked from https://github.com/aeternity/rebar3/commit/a8b639251a53744485e9689f62b105e524d322b2 i.e. PR https://github.com/aeternity/rebar3/pull/3

### CODE REVIEW

**Review by tolbrino — APPROVED** [review]

File: src/rebar_prv_escriptize.erl [hunk 1]
```diff
@@ -80,11 +80,16 @@ do(State) ->
                     ?PRV_ERROR({bad_name, Name})
             end
     end,
-    AppInfo1 = rebar_hooks:run_all_hooks(Cwd, pre, ?PROVIDER, Providers, AppInfo0, State),
-    ?INFO("Building escript...", []),
-    Res = escriptize(State, AppInfo1),
-    rebar_hooks:run_all_hooks(Cwd, post, ?PROVIDER, Providers, AppInfo1, State),
-    Res.
+    case AppInfo0 of
```
Comment (lucafavatella) [inline src/rebar_prv_escriptize.erl:83]: I am trying to understand if this is meant to go upstream too.

Comment (lucafavatella) [reply src/rebar_prv_escriptize.erl:83]: I see this already proposed (and merged!) upstream. Thanks.

https://github.com/erlang/rebar3/pull/2027/commits/93e300b7657d34111f11257e23aebb629ef21438

**Review by lucafavatella — COMMENTED** [review]
Changes by Tino are ok to me.

File: src/rebar_prv_escriptize.erl (hunk 1 above)
Comment (tolbrino) [reply src/rebar_prv_escriptize.erl:83]: This is already landed in upstream master. I backported it here, until the next rebar3 release is being used.

**Review by dincho — APPROVED** [review]
CI configuration looks OK.

**Pull request merged**